test/unit/services/appealService.test.js

```diff
@@ -0,0 +1,189 @@
+const { getAppeal, changeEmailAddress, stopReceivingEmails } = require('app/services/appealService');
+const { expect, sinon } = require('test/chai-sinon');
+const nock = require('nock');
+const { appealsAPI } = require('app/config');
+const HttpStatus = require('http-status-codes');
+const mockAppeal = require('test/mock/data').md002.appeal;
+
+describe('appealService.js', () => {
+
+  const invalidId = 'invalidId';
+  let req, res, next, api;
+
+  beforeEach(() => {
+
+    req = {
+      params: {},
+      body: {
+        email: 'myemail@email.com'
+      }
+    };
+
+    res = {
+      locals: {
+        appeal: {},
+        token: {
+          appealId: 'md002',
+          subscriptionId: 'qwerty123'
+        }
+      }
+    };
+
+    next = sinon.stub();
+
+  });
+
+  afterEach(() => {
+    api.isDone();
+    res.locals.appeal = {};
+  });
+
+  describe('getAppeal', () => {
+
+    describe('get request to api is successful', () => {
+
+      it('should call next', () => {
+
+        const appeal = mockAppeal;
```

Comment:
There's no need to reference `mockAppeal` from another constant `appeal` we can use the first one.